### PR TITLE
Fixes some cell's display & formatting issues 

### PR DESCRIPTION
**Pull request by stel**

### CODE REVIEW

File: RealmBrowser/Controllers/RLMInstanceTableViewController.m [hunk 1]
```diff
@@ -360,14 +353,14 @@ -(NSView *)tableView:(NSTableView *)tableView viewForTableColumn:(NSTableColumn
...
-                numberCellView.textField.delegate = self;
                 numberCellView.textField.target = self;
                 numberCellView.textField.action = @selector(editedTextField:);
             }
 
-            numberCellView.textField.stringValue = [realmDescriptions printablePropertyValue:propertyValue ofType:type];
-
-            ((RLMNumberTextField *)numberCellView.textField).number = propertyValue;
+            // Lets move all formatting stuff to cell itself
+            // numberCellView.textField.stringValue = [realmDescriptions printablePropertyValue:propertyValue ofType:type];
```
Comment (jpsim) [inline RealmBrowser/Controllers/RLMInstanceTableViewController.m]: these comments don't really add value unless you know what we were doing before, which is what git history is for 😉

**Comment by jpsim** [comment]
👍

**Comment by stel** [comment]
Finished some layout & formatting fixes for cells and sidebar, looks like it's ready for merging.

/cc @TimOliver @jpsim

**Comment by TimOliver** [comment]
Awesome! Thanks a lot for that Dmitry! :)

**Pull request merged**